modules/mobile/spec/request/v1/immunizations_request_spec.rb

```diff
@@ -40,331 +40,52 @@
-                         'www.example.com/mobile/v0/health/locations/I2-3JYDMXC6RXTU4H25KRVXATSEJQ000000' } } } },
-             { 'id' => 'I2-N7A6Q5AU6W5C6O4O7QEDZ3SJXM000000',
-               'type' => 'immunization',
-               'attributes' =>
-                 { 'cvxCode' => 207,
-                   'date' => '2020-12-18T12:24:55Z',
-                   'doseNumber' => nil,
-                   'doseSeries' => nil,
-                   'groupName' => nil,
-                   'manufacturer' => nil,
-                   'note' =>
-                     'Dose #1 of 2 of COVID-19, mRNA, LNP-S, PF, 100 mcg/ 0.5 mL dose vaccine administered.',
-                   'reaction' => nil,
-                   'shortDescription' => 'COVID-19, mRNA, LNP-S, PF, 100 mcg/ 0.5 mL dose' },
-               'relationships' =>
-                 { 'location' =>
-                   { 'data' =>
-                     { 'id' => 'I2-3JYDMXC6RXTU4H25KRVXATSEJQ000000', 'type' => 'location' },
-                     'links' =>
-                       { 'related' =>
-                         'www.example.com/mobile/v0/health/locations/I2-3JYDMXC6RXTU4H25KRVXATSEJQ000000' } } } },
-             { 'id' => 'I2-2BCP5BAI6N7NQSAPSVIJ6INQ4A000000',
-               'type' => 'immunization',
-               'attributes' =>
-                 { 'cvxCode' => 207,
-                   'date' => '2021-01-14T09:30:21Z',
-                   'doseNumber' => nil,
-                   'doseSeries' => nil,
-                   'groupName' => nil,
-                   'manufacturer' => nil,
-                   'note' =>
-                     'Dose #2 of 2 of COVID-19, mRNA, LNP-S, PF, 100 mcg/ 0.5 mL dose vaccine administered.',
-                   'reaction' => nil,
-                   'shortDescription' => 'COVID-19, mRNA, LNP-S, PF, 100 mcg/ 0.5 mL dose' },
-               'relationships' =>
-                 { 'location' =>
-                   { 'data' =>
-                     { 'id' => 'I2-3JYDMXC6RXTU4H25KRVXATSEJQ000000', 'type' => 'location' },
-                     'links' =>
-                       { 'related' =>
-                         'www.example.com/mobile/v0/health/locations/I2-3JYDMXC6RXTU4H25KRVXATSEJQ000000' } } } }],
-            'meta' => {
-              'pagination' => {
-                'currentPage' => 1,
-                'perPage' => 100,
-                'totalPages' => 1,
-                'totalEntries' => 15
-              }
-            },
-            'links' => {
-              'self' => 'http://www.example.com/mobile/v1/health/immunizations?page[size]=100&page[number]=1',
-              'first' => 'http://www.example.com/mobile/v1/health/immunizations?page[size]=100&page[number]=1',
-              'prev' => nil,
-              'next' => nil,
-              'last' => 'http://www.example.com/mobile/v1/health/immunizations?page[size]=100&page[number]=1'
-            } }
-        )
-      end
-
-      context 'for items that do not have locations' do
```

Comment:
Github's diffing is making this difficult to read. I removed the "matches expected attributes" tests for the "has/has not location" tests because we're essentially testing that elsewhere.